### DISCUSSION TITLE
How is places data curated and verified

### DISCUSSION THREAD

**Discussion by songololo** [OvertureMaps/data#102] [Discussion]
I'm curious if the logic for the generation of places data explained anywhere, such as how the data is curated and verified?

To what extent does it overlap OSM and how are OSM places deemed of sufficient quality to be included? Many OSM places are not included in the Overture dataset but I'm not clear on what the related logic involved is.

**Comment by D3M05** [comment]
I'm not affiliated with the providers, but as far as I know, the Places data isn't related to the OSM data in any way.  OSM's database license requires that any database that uses it, even if just to compare or remove duplicate data, must then also have a similar license with all data available upon request.  That would change the current Places license, which is different to the other licenses that do use OSM.  
Keeping the current license is important, because it allows people to integrate it with databases that are incapable of being made public due to restrictions by their creators.  I'm guessing this is probably similar reasoning for Meta and Microsoft who also pay for proprietary databases.  Obviously this doesn't keep them from making Places open data, and my company has already freely offered improvements to the data.  Personally, I wish the OSM board would open up the licensing to at least allow comparison for de-duplication sake, but last I've checked they only partially allow it in geocoding.

**Reply by songololo** [reply]
Thanks, this is good to know.
